rollup.base.config.js

```diff
@@ -226,21 +227,47 @@ function getSourceMapPathTransformer(distPath, theNameSpace, isDebug) {
                 console.log(` -- Absolute: ${absPath}`);
             }
 
-            let idx = absPath.indexOf("node_modules");
-            if (idx != -1) {
-                if (isDebug) {
-                    console.log(` -- NodeModule: ${absPath}`);
+            // Special handling for @nevware21 packages that may be referenced in sourcemaps
+            if (normalizedPath.indexOf("@nevware21/") !== -1 && (normalizedPath.indexOf("build/es5/") !== -1 || normalizedPath.indexOf("/mod/") !== -1)) {
+                const packageName = normalizedPath.match(/@nevware21\/([^\/]+)/)?.[0];
```

Comment:
Is this still required as the @nevware21 projects now include map file references that link back to the specifically github repo with the tagged release number